CHANGES.rst

```diff
@@ -56,6 +56,9 @@ cadc
 
 - Deprecated keywords and ``run_query`` method have been removed. [#2389]
 
+- Fixed a bug to be able to pass longer that filename Path objects as
+  ``output_file``. [#2541]
+
```

Comment:
I don't know if this was a bug. The documentation for the method states that `output_file` is either a string or a file handler. Adding support for `Path` in addition to that is a great idea but it's a new feature.